docs/user-guide.rst

```diff
@@ -3,36 +3,249 @@
+   troi playlist --print daily-jams <user_name>
 
 This will generate a playlist and print the list to the terminal.
 
 
-To generate a playlist from a patch and display it and then upload it to ListenBrainz:
+To generate a global playlist from a patch and display it and then upload it to ListenBrainz:
 
 .. code-block:: bash
 
-   python -m troi.cli playlist --print --upload --token <user-token> daily-jams <user_name>
+   troi playlist --print --upload --token <user-token> daily-jams <user_name>
 
 This will generate a playlist and print the list to the terminal and then upload it to ListenBrainz. You can find your
 user token on your `profile page at ListenBrainz <https://listenbrainz.org/profile/>`_.
 
 From here on you can explore different :ref:`patches` or read how Troi works :ref:`technical-introduction`
+
+
+Local Playlist Generation
+-------------------------
+
+Local playlists are playlists that have been resolved against a local collection
+and are playable to a local player. In order for this to work, you will need
+to index your local collection using the database tools first. 
+
+Index your music collection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**IMPORTANT**: This feature only works if you music collection is tagged with MusicBrainz tags. We recommend
+`Picard <http://picard.musicbrainz.org>`_ for tagging your collection.
+
+If you're unwilling to properly tag your collection, then please do not contact us to request that we remove
+this requirement. We can't. We won't. Please close this tab and move on.
+
+If you have your collection hosted on an app like Funkwhale, Navidrome or Gonic, who have a Subsonic API, you can generate
+playlists directly the web application. Alternatively, if you music collection isn't available via a Subsonic API, you can
+scan a local collection of files and make local m3u playlists.
+
+*Note*: We recommend that you scan *either* a local filesystem collection or a subsonic API hosted collection. Doing both
+is going to result in erratic behaviour of the content resolver.
+
+Setting up config.py
+""""""""""""""""""""
+
+While it isn't strictly necessary to setup **config.py**, it makes using the troi content resolver easier:
+
+.. code-block:: bash
+
+    cp config.py.sample config.py
+
+Then edit **config.py** and set the location of where you're going to store your resolver database file
+into **DATABASE_FILE**. If you plan to use a Subsonic API, then fill out the Subsonic section as well.
+
+If you decide not to use the **config.py** file, make sure to pass the path to the DB file with **-d** to each
+command. All further examples in this file assume you added the config file and will therefore omit  
+the **-d** option.
+
+You can also define a set of directories to be used by default for the scan command in **MUSIC_DIRECTORIES**.
+
+Options for saving playlists
+""""""""""""""""""""""""""""
+
+The playlist generation functions below print the generated playlist and nothig else. To save the playlist you'll
```

Comment:
This is fine for a first release but in future we can autogenerate this from click commands help itself. We do it in LB too, maybe open a ticket?

Reply:
Fixed, by creating a whole new page.